src/insert/spacefinder/article.ts

```diff
@@ -293,10 +270,6 @@ const init = async (fillAdSlot: FillAdSlot): Promise<void> => {
 	// We add the first inline ad before finding a place for an im so that the inline1 doesn't get pushed down
 	await addFirstInlineAd(fillAdSlot);
 
-	if (window.guardian.config.page.hasInlineMerchandise) {
-		await attemptToAddInlineMerchAd(fillAdSlot);
-	}
-
 	// Add the other inline slots after the im, so that there is space left for the im ad
 	await addSubsequentInlineAds(fillAdSlot);
```

Comment:
I think the first and subsequent passes can be combined again now

Reply:
The consentless code also uses this `addSubsequentInlineAds` function, and we need to keep the first and subsequent passes separate to fix the bug I'm currently looking at. Could we keep them separate for now?

Reply:
Yes like Emma said it's used by consentless so I think keeping them separate for now is a good idea until a refactor in a separate PR maybe.

Reply:
Ok I've done some testing, I was wrong and we can merge these back together. I will pick it up as a separate follow up PR so that we don't delay releasing this!

Reply:
Ok Emma sounds like a plan. Thank you!